doc/nrf/libraries/bin/lwm2m_carrier/CHANGELOG.rst

```diff
@@ -9,6 +9,59 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+liblwm2m_carrier 3.3.0
+**********************
+
+Release for modem firmware version 1.3.5 and 2.0.0
+
+Certification status
+====================
+
+For certification status, see `Mobile network operator certifications`_.
+
+Size
+====
+
+See :ref:`lwm2m_lib_size` for an explanation of the library size in different scenarios.
+
++-------------------------+---------------+------------+
+|                         | Flash (Bytes) | RAM (Bytes)|
++-------------------------+---------------+------------+
+| Library size            |               |            |
+| (binary)                |               |            |
++-------------------------+---------------+------------+
+| Library size            |               |            |
+| (reference application) |               |            |
++-------------------------+---------------+------------+
+
+Changes
+=======
+
+* Added the Binary App Data Container object.
+
+  * The object is enabled for the SoftBank subscriber ID :kconfig:option:`CONFIG_LWM2M_CARRIER_SOFTBANK`, or if the LwM2M carrier library is operating in generic mode :kconfig:option:`CONFIG_LWM2M_CARRIER_GENERIC`.
+
+* Added the Event Log object.
+
+  * The object is enabled for the SoftBank subscriber ID :kconfig:option:`CONFIG_LWM2M_CARRIER_SOFTBANK`, or if the LwM2M carrier library is operating in generic mode :kconfig:option:`CONFIG_LWM2M_CARRIER_GENERIC`.
+
+* FOTA procedure now only requires a single modem library initialization. For more information see the modem library :ref:`nrf_modem_changelog`.
+
+  * Old successful event flow: :c:member:`LWM2M_CARRIER_EVENT_FOTA_START` > :c:member:`LWM2M_CARRIER_EVENT_LTE_POWER_OFF` > :c:member:`LWM2M_CARRIER_EVENT_MODEM_SHUTDOWN` > :c:member:`LWM2M_CARRIER_EVENT_MODEM_INIT` > :c:member:`LWM2M_CARRIER_EVENT_MODEM_INIT` > :c:member:`LWM2M_CARRIER_EVENT_FOTA_SUCCESS` > :c:member:`LWM2M_CARRIER_EVENT_LTE_LINK_UP`.
+  * New successful event flow: :c:member:`LWM2M_CARRIER_EVENT_FOTA_START` > :c:member:`LWM2M_CARRIER_EVENT_LTE_POWER_OFF` > :c:member:`LWM2M_CARRIER_EVENT_MODEM_SHUTDOWN` > :c:member:`LWM2M_CARRIER_EVENT_MODEM_INIT` > :c:member:`LWM2M_CARRIER_EVENT_FOTA_SUCCESS` > :c:member:`LWM2M_CARRIER_EVENT_LTE_LINK_UP`.
+
+AT&T support has been deprecated
+--------------------------------
+
+The application shall no longer connect to the AT&T Device Management server.
```

Comment:
```suggestion
The application no longer connects to the AT&T Device Management server.
```

Reply:
In this case, that LwM2M carrier library _can_ in fact connect to the server. However the customer application should (shall) no longer do it because of changes in requirements from AT&T. More context is given in the following bullet points, so I would leave it as it is now.

Reply:
Then we could say: The application must no longer connect to the AT&T Device Management server.

This is to avoid using word "shall".